templates/etc/haproxy/frontend.cfg.j2

```diff
@@ -86,7 +93,9 @@ frontend {{ frontend.name }}
 {% endif %}
 {% endif %}
 
+{% if frontend.default_backend is defined %}
```

Comment:
Can you explain why this is needed?

Reply:
In certain configurations eg. if *:80 acts only as redirector to *:443, backend is not needed.